### ISSUE TITLE
Needs auto scrolling.

### ISSUE THREAD

**Issue by Goretantath** [parsiad/nexus-autodl#48]
Wabajak sometimes doesn't auto scroll to the download button so you have to manualy scroll down and click the button on the window it opens up. If you could somehow implement an autoscroll after a certain amount of time it hasnt detected the button thatd prob fix this. Thanks in advance!

**Comment by orn8** [comment]
agreed. i use an [autohotkey](https://www.autohotkey.com/) v1.1 script i edited from someone to fix my issue.

```ahk
f7:: Settimer,Scroll,2000 ; milliseconds how long before each scroll
f8:: SetTimer,Scroll, off

f9::
msgbox, Exiting...
Exitapp

scroll:
  send, {WheelDown 1} ; how much to scroll each time
return
```

f7 to start, f8 to stop, f9 to exit the script. this script automatically scrolls down 1 unit on the scroll wheel every 2 seconds. use this script with nexus-autodl's min and max pause time set to exactly 1 or lower.

i would appreciate improvemnts to this autohotkey script, as i essentially just pulled it off [this thread](https://www.autohotkey.com/boards/viewtopic.php?t=27522).
